src/main/java/io/github/perrymant/moneymaker/FileIO.java

```diff
@@ -0,0 +1,31 @@
+package io.github.perrymant.moneymaker;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+class FileIO {
+
+    static String read(String fileName) throws IOException {
```

Comment:
If you've got `throws BlahException` in a method declaration that's a sign that you're doing something wrong. Remember: avoid checked exception - always wrap in (preferably your own implementation of a) RuntimeException.